client/reader/follow-button/index.jsx

```diff
@@ -9,4 +9,4 @@ import config from 'config';
 import FluxButton from './flux';
 import ReduxButton from './redux';
 
-module.exports = config.isEnabled( 'reader/following-manage-refresh' ) ? ReduxButton : FluxButton;
+export default ( config.isEnabled( 'reader/following-manage-refresh' ) ? ReduxButton : FluxButton );
```

Comment:
ahhh, so the set of exports cannot be conditional, but what is assigned to those exports can be. subtle.